pandas/core/indexes/category.py

```diff
@@ -354,7 +354,7 @@ def _to_safe_for_reshape(self):
 
     def get_loc(self, key, method=None):
         """
-        Get integer location for requested label
+        Get integer location for requested label.
```

Comment:
I actually like the "Get integer location, slice or boolean mask for requested label" from the multi-index docstring, as this seems to be more complete description?

Reply:
I agree, Will change